_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,75 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Talks
+
+We visited several talks and three workshops:
+
+## Reactive Programming
+
+[A Reactive Take on Pokémon GO](https://programm.javaland.eu/2017/talk.html#talk?talkId=529316) by [Riccardo Lippolis](https://www.linkedin.com/in/rlippolis) and [Bas Knopper](https://twitter.com/BWknopper) was all about [Spring Reactive](http://docs.spring.io/spring-framework/docs/5.0.0.M1/spring-framework-reference/html/web-reactive.html). After a quick introduction he showed how to build a reactive program for smartphones.
+He extended the observer pattern and added push-based streams, which can be used when it is unknown when data is available.
+At its core the [Spring Reactor](fully non-blocking foundation with efficient demand management), a fully non-blocking foundation with efficient demand management, is used.
+Another important role plays [Flux](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html) which is similar to the [Java 8 Stream API](https://docs.oracle.com/javase/8/docs/api/java/util/stream/package-summary.html).
+A new paradigm in Spring Land will be released in June and is called [Spring Web Flux](https://spring.io/blog/2017/03/15/spring-tips-the-spring-web-flux-reactive-client).
+See [this video](https://www.youtube.com/watch?v=leZdgr-O4LE) by [Josh Long](https://twitter.com/starbuxman) for more details.
+
+## Migrating to Java 9
+
+In [this talk](https://programm.javaland.eu/2017/talk.html#talk?talkId=529430) by [Sander Mak](https://twitter.com/sander_mak), who is currently writing a book with [Paul Bakker](https://twitter.com/pbakker) about [Java 9 Modularity](https://twitter.com/javamodularity), explained in this talk the essential steps to prepare for the next Java major release.
+Use [jdeps](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/jdeps.html) to get information about missing platform modules.
+This tool has some interesting option to inspect JDK internals like and some you should not use like `—add-exports` and `—permit-illegal-access `.
+Better change your could instead of using these flags.
+If you want to migrate to use modules you can put a plain jair on the module path (automatic modules).
+The name is derived from the jar name and export everthing needed as well as reads other downstream modules.
+It can access the class path. You should create a module which requires `commons.lang` and a `module-info.jar`.
+Dependencies have to be module themselves for easier handling (transitive dependencies).
+At runtime services (components) are accessible via reflection.
+Don’t open module to everyone.
+Export only certain api packages including services!
+Revisit open modules when you are done.
+Create several modules that are logically seperated, e.g. `API`, `Impl` and `Main`.
+
```

Comment:
The lines above sound like bulle tpoints. 
I think you either use more connectors like "For that reason", "Moreover", etc. or you intentionally put the above sentences in a list with bullet points